Lib/fontTools/ufoLib/glifLib.py

```diff
@@ -12,19 +12,21 @@
 
 from __future__ import annotations
 
-import enum
 import logging
 from collections import OrderedDict
+from typing import TYPE_CHECKING, cast, Any, Optional, Union
+from lxml import etree
```

Comment:
we don't want to unconditionally import lxml, it's an optional extra, that's why we have fontTools.misc.etree module, to fall back to the stdlib ElementTree when lxml is not installed!